### ISSUE TITLE
Tokens and Avatars not rendering properly. 

### ISSUE THREAD

**Issue by ExW-Tritone** [MrPrimate/vtta-tokenizer#4]
Posted in the discord as well

I am having an issue with this module. I've tested with a fresh world and it seems to work as intended, but on my current game world, even with all modules turned off It's not working on the main world that I play on. 

Here is the console that I get from opening to saving. 

```
Foundry VTT | Rendering ModuleManagement
foundry.js:5964 Foundry VTT | Retrieved and compiled template templates/sidebar/apps/module-management.html
foundry.js:4503 Foundry VTT | Rendering SettingsConfig
foundry.js:5964 Foundry VTT | Retrieved and compiled template templates/sidebar/apps/settings-config.html
DirectoryPicker.js:84 Adding Picker Button
foundry.js:4503 Foundry VTT | Rendering Tokenizer
index.js:111 Setting Avatar dimensions to 512/512
DirectoryPicker.js:42 Uncaught (in promise) TypeError: Cannot read property 'match' of null
    at Function.parse (DirectoryPicker.js:42)
    at Function.uploadToFoundryV3 (utils.js:162)
    at index.js:72
```
Here's a gif replicating the issue.

![BugReport Tokenizer](https://user-images.githubusercontent.com/69099105/99438139-61e3be00-28e1-11eb-9ad4-3cac6772bc5a.gif)

**Comment by BenjaMan22** [comment]
static parse(str) {
    let matches = str.match(/\[(.+)\]\s*(.+)/);
    if (matches) {

str is null on any token I try to save.

**Comment by elfonochasis** [comment]
I've the same problem with my actual world.

**Comment by elizeuangelo** [comment]
Solved this by creating an actual tokens folders instead of saving in foundry/data.

In previous versions when the settings for the folder was empty, it would save as a string "". Now it saves as null and it bugs the module.

**Comment by MrPrimate** [comment]
Should be fixed in 2.2.4
